prac_06/guitars.py

```diff
@@ -0,0 +1,56 @@
+"""
+Ashton Jack Stewart
+Practical 06 - guitars
+"""
```

Comment:
You needed to guess how long it would take you to complete and how long it actually took you.